Dockerfile.windows

```diff
@@ -0,0 +1,57 @@
+# This file describes the standard way to build Docker, using docker on TP4
+#
+# Usage:
+#
+# # Assemble the full dev environment. This is slow the first time.
+# docker build -t docker -f Dockerfile.windows .
+#
+# # Mount your source in an interactive container for quick testing:
+# docker run -it -v ${pwd}\bundles:c:/go/src/github.com/docker/docker/bundles docker cmd
+#
+# # Build the docker binary in bundles\docker.exe
+# docker run -it -v ${pwd}\bundles:c:/go/src/github.com/docker/docker/bundles docker "powershell -file hack\make.ps1"
+#
+
+FROM windowsservercore
+
+ENV GOLANG_VERSION 1.5.1
+
+RUN mkdir \go\bin \go\src
+WORKDIR /go
+ENV GOPATH C:/go;C:/go/src/github.com/docker/docker/vendor
+RUN setx PATH %PATH%;C:\PortableGit\bin;C:\PortableGit\usr\bin;C:\tools\go\bin;C:\go\bin
+
+RUN powershell -Command \
+    Sleep 2 ; \
+    iex ((new-object net.webclient).DownloadString('https://chocolatey.org/install.ps1')) ; \
+    choco install -y golang -version %GOLANG_VERSION% ; \
+    wget https://github.com/git-for-windows/git/releases/download/v2.6.3.windows.1/PortableGit-2.6.3-64-bit.7z.exe -OutFile \gitinstaller.exe ; \
```

Comment:
hmm alright, I don't know much about it, I think it's just an alias. If it's a short alias, then we should probably use the canonical command instead.